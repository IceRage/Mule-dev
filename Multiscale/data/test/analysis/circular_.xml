<?xml version="1.0" encoding="utf-8"?>
<!--Warning! This xml file was automatically generated by a C++ program using the Boost PropertyTree library.-->
<experiment>
	<timepoint>
		<spatialEntity spatialType="region">
			<clusteredness>0.9569923038859586</clusteredness>
			<density>0.4639648561294609</density>
			<area>45449</area>
			<perimeter>1672.32129073143</perimeter>
			<distanceFromOrigin>270.7120241141867</distanceFromOrigin>
			<angle>83.29555332647426</angle>
			<triangleMeasure>0.601032903390844</triangleMeasure>
			<rectangleMeasure>0.4747973807218359</rectangleMeasure>
			<circleMeasure>0.2874802089404139</circleMeasure>
			<centroidX>382.9199</centroidX>
			<centroidY>983.9192</centroidY>
		</spatialEntity>
		<spatialEntity spatialType="region">
			<clusteredness>1</clusteredness>
			<density>0.553767539844759</density>
			<area>37028</area>
			<perimeter>1197.486457109451</perimeter>
			<distanceFromOrigin>105.152270541344</distanceFromOrigin>
			<angle>32.62964665912187</angle>
			<triangleMeasure>0.6528327438957535</triangleMeasure>
			<rectangleMeasure>0.542308898571211</rectangleMeasure>
			<circleMeasure>0.238980646090803</circleMeasure>
			<centroidX>872.082</centroidX>
			<centroidY>354.9841</centroidY>
		</spatialEntity>
		<numericStateVariable>
			<name>avgClusterednessRegions</name>
			<value>0.001255070714248408</value>
		</numericStateVariable>
		<numericStateVariable>
			<name>avgDensityRegions</name>
			<value>0.50886619798711</value>
		</numericStateVariable>
	</timepoint>
</experiment>
