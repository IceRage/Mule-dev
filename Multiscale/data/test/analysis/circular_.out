Clusteredness degree,Density,Area,Perimeter,Distance from origin,Angle(degrees),Shape,Triangle measure,Rectangle measure,Circle measure,Centre (x-coord),Centre (y-coord)
0.956992,0.463965,45449,1672.32,270.712,83.2956,triangular,0.601033,0.474797,0.28748,382.92,983.919
1,0.553768,37028,1197.49,105.152,32.6296,triangular,0.652833,0.542309,0.238981,872.082,354.984

Average clusteredness degree: 0.00125507
Average density: 0.508866
